src/recursive_sorting/recursive_sorting.py

```diff
@@ -6,9 +6,25 @@ def merge( arrA, arrB ):
     i = j = 0
     index = 0
     while i < len(arrA) and j < len(arrB):
```

Comment:
Instead of having multiple while loops, your code can be simplified into one single for loop like this: 

```
for x in range(elements):
    if i >= len(arrA):
      merged_arr[x] = arrB[j]
      j += 1
    elif j >= len(arrB):
      merged_arr[x] = arrA[i]
      i += 1
    elif arrA[i] > arrB[j]:
      merged_arr[x] = arrB[j]
      j += 1
    else:
      merged_arr[x] = arrA[i]
      i += 1
```